tonic/src/codec/decode.rs

```diff
@@ -32,7 +35,10 @@ impl<T> Unpin for Streaming<T> {}
 #[derive(Debug)]
 enum State {
     ReadHeader,
-    ReadBody { compression: bool, len: usize },
+    ReadBody {
+        is_compressed: bool,
+        message_length: usize,
```

Comment:
I think this follows rust a bit better :)

```suggestion
        message_len: usize,
```